### ISSUE TITLE
Feature request: measure the output

### ISSUE THREAD

**Issue by ridercz** [xypwn/scadqr#9]
I propose new API:

```scad
module qr_size(message, error_correction = "M", encoding = "UTF-8")
```

This would return a scalar number, which is the width and height of QR code in modules. 

If the model is to be 3D printed, there is a minimal printable module size (different for each printer, based on technology used and other factors like nozzle diameter for FDM printers).

Currently the API works by the caller providing the dimensions and the code fitting within them. This can work for some scenarios, but does not for many others. It can lead to unprintable models, if the size is small and data amount in code big and there is no way how the caller can know the code size to count with the density. It would be nice to be able to get size of the code, in modules, based on content. Therefore the physical size can be set to be printable and other dimensions can be computed from that.

Imagine the simplest possible example: 3D printed badge with QR code. The size of the badge must depend on amount of data stored on QR code. At this time such model cannot be made in universal parametric way. With the new API it would be possible.

**Comment by xypwn** [comment]
Great suggestion!

Implementing it should be very simple, since it's basically already implemented in the main qr code generation function: https://github.com/xypwn/scadqr/blob/main/src/qr.scad#L46-L49 .

I'll see to it in the next couple of days whenever I find the time.

**Comment by ridercz** [comment]
Great. I thought it would be somewhere and probably could do it myself in scad file, but I don't know Python and you use it somehow to build the scad file, so I leave it up to you :)

**Comment by xypwn** [comment]
You don't have to know python! All Python does here is build the monolithic source file and run tests (all documented in the Development section of the README). Of course, I'd appreciate a PR, but you don't have to :)

**Comment by ridercz** [comment]
Oh, now I see what and why you are doing. That's clever, I use it. You have PR now.

**Comment by xypwn** [comment]
Implemented in #11
